src/constants/constants.ts

```diff
@@ -296,3 +296,45 @@ export const x64 = "x64";
 export const windowsDockerDesktopExecutable = "Docker Desktop.exe";
 export const docker = "docker";
 export const dockerDeploymentLoggerChannelName = "Docker Deployment";
+
+// SQL Database Projects Constants
+export const DSP_PREFIX = "Microsoft.Data.Tools.Schema.Sql.Sql";
+export const DSP_SUFFIX = "DatabaseSchemaProvider";
+export const sqlServerDockerRegistry = "mcr.microsoft.com";
+export const sqlServerDockerRepository = "mssql/server";
+export const sqlServerEulaLink = "https://aka.ms/mssql-container-license";
+export const licenseAgreementUrl =
+    "https://github.com/microsoft/containerregistry/blob/main/legal/Container-Images-Legal-Notice.md";
+export const dockerImageDefaultTag = "latest";
+export const AzureSqlDbFullDockerImageName = "Azure SQL Database (Edge emulation)";
```

Comment:
Should this (and the other image names) be localized?  Might need confirmation with PM because it's branding